Sources/SwiftDocC/Model/Rendering/Symbol/TaskGroupRenderSection.swift

```diff
@@ -86,11 +86,11 @@ public struct TaskGroupRenderSection: RenderSection, Equatable {
 extension TaskGroupRenderSection {
 
     /// Creates a new instance with the given automatically curated task group data.
-    init(taskGroup group: AutomaticCuration.TaskGroup) {
+    init(taskGroup group: AutomaticCuration.TaskGroup, generated: Bool) {
```

Comment:
I'm not 100% sure, but reading this code it seems like the extension is adding an initializer that will only be called when the task group is an instance of `AutomaticCuration.TaskGroup`. In that case, wouldn't we always want generated=true?

Should we remove the generated parameter and instead just call `self.generated = true` below?

Probably I am misunderstanding something?

Reply:
You're right. Thanks for catching that. I changed this to always set `self.generated = true` and updated a few more places in the code to call this instead of initializing the `TaskGroupRenderSection` per member explicitly.